src/layman/layer/util.py

```diff
@@ -220,12 +244,26 @@ def abort_layer_chain(workspace, layername):
     celery_util.abort_publication_chain(workspace, LAYER_TYPE, layername)
 
 
+def abort_layer_chain_by_uuid(uuid):
+    info = layman_util.get_publication_info_by_uuid(uuid, context={'keys': ['workspace', 'name']})
+    workspace = info['_workspace']
+    layername = info['name']
+    return abort_layer_chain(workspace, layername)
+
+
 def is_layer_chain_ready(workspace, layername):
     chain_info = get_layer_chain(workspace, layername)
     return chain_info is None or celery_util.is_chain_ready(chain_info)
 
 
-lock_decorator = redis_util.create_lock_decorator(LAYER_TYPE, 'layername', is_layer_chain_ready)
+def is_layer_chain_ready_by_uuid(uuid):
```

Comment:
It would be nice to make one general is_publication_chain_ready_by_uuid instead of two `is_layer_chain_ready_by_uuid` and `is_map_chain_ready_by_uuid`. But it requires some bigger changes, so not in this PR :-)